src/payment/strategies/paypal-commerce/paypal-commerce-script-loader.ts

```diff
@@ -0,0 +1,32 @@
+import { ScriptLoader } from '@bigcommerce/script-loader';
+
+import { InvalidArgumentError } from '../../../common/error/errors';
+import { PaymentMethodClientUnavailableError } from '../../errors';
+
+import { PaypalCommerceHostWindow, PaypalCommerceSDK } from './paypal-commerce-sdk';
+
+export default class PaypalCommerceScriptLoader {
+    private _window: PaypalCommerceHostWindow;
+
+    constructor(
+        private _scriptLoader: ScriptLoader
+    ) {
+        this._window = window;
+    }
+
+    async loadPaypalCommerce(clientId: string, currency: string = 'USD'): Promise<PaypalCommerceSDK> {
+        if (!clientId) {
+            throw new InvalidArgumentError();
+        }
+
+        const scriptSrc = `https://www.paypal.com/sdk/js?currency=${currency}&client-id=${clientId}`;
```

Comment:
@helga1507 can we also pass locale https://developer.paypal.com/docs/checkout/reference/customize-sdk/#locale, to the script params, 
I think it would be better to have an object with parameters, instead of arguments (because there could be a lot arguments)

Reply:
@helga1507 , I am creating separate tickets for additional params that need to be passed, it would be out of the scope of this PR
I think we should add an object that would have all properties for script initialization for future